### ISSUE TITLE
Unifying fips related files

### ISSUE THREAD

**Issue by cagilerbas** [floooh/fips#177]
Currently fips creates multiple files and folders (like .fips-gen.py, fips-configs etc) within a project directory. Is this arrangement intentional for a purpose, wouldn't it be better to collect all fips related construct within one special directory?

**Comment by floooh** [comment]
Yes, I agree. It's on my mental todo list, but never pressing enough to do something about it...

The idea is to have an optional sub-directory in a project called something like ```fips-files/```, and all current fips-related files and directories (with the exception of the ```fips``` script itself, and maybe the fips.yml file) would be there, e.g.

```
fips-files/
    configs/
    verbs/
    generators/
    include.cmake/
    ...
```
The old structure would also still need to work though, so the currently hardwired paths in fips would need to be replaced with function calls which return one path or another.

**Comment by floooh** [comment]
I have committed a first version of this just now. The old project structure still works (and will continue to work), but the main fips directories and fips-include.cmake can now be grouped under a single 'fips-files' directory. This is a first step, I may move more files in the future, but the old project directory structure will always work.

I'll keep this ticket open until I have updated the documentation.
